src/status_im/contexts/wallet/send/routes/style.cljs

```diff
@@ -25,20 +21,26 @@
 (def section-label-left
   {:width 136})
 
-(def network-link
+(def network-links-container
   {:margin-horizontal -1.5
-   :z-index           1
+   :margin-top        7.5
+   :z-index           10
```

Comment:
Why `z-index` 10? Probably 10 is not needed. If it needs a `z-index` higher than 1, then better check the exact value it needs which is probably 2 or 3. This avoids having `z-indexes` like `100` `999` when the component needs updates

Reply:
Done